src/test/java/org/cactoos/iterator/IteratorOfTest.java

```diff
@@ -70,9 +77,15 @@ public void nonEmptyIteratorDoesNotHaveNext() {
 
     @Test
     public void nonEmptyIteratorThrowsException() {
+        final Iterator<Character> iterator = new Skipped<>(
```

Comment:
@fanifieiev please do not rely on `Skipped` here, it's too far away from the tested class to make the test robust

Reply:
@victornoel Agree, made the fix